integ/scripts/bash/rfdk-integ-e2e.sh

```diff
@@ -42,6 +42,9 @@ mkdir -p $INTEG_TEMP_DIR
 # Stage deadline from script
 $BASH_SCRIPTS/stage-deadline.sh
 
+# Extract the Deadline version to use for Deadline installations on the farm. This will override any other Deadline version provided.
+export DEADLINE_VERSION=$(node -e $'const json = require(process.argv[1] + \'/manifest.json\'); console.log(json.version)' "$DEADLINE_STAGING_PATH")
```

Comment:
My understanding of the purpose of this is code is to fully-resolve the `DEADLINE_VERSION` after it has been staged. We need the fully-resolved version to determine the AWS Portal AMI IDs programatically. To do this, we are getting the resolved version from the stage manifest. So if someone running the tests set `DEADLINE_VERSION` initially to `10.1.11`, we can look at the manifest and get the full version (e.g. `10.1.11.5`) and use that to find the corresponding AMI IDs.

If this is correct, can we update the comment to explain this? Otherwise, it's not really clear why we are doing this.

Reply:
You are right.

Reply:
right. looks good now. resolved